R/lazy-data.R

```diff
@@ -0,0 +1,26 @@
+#' @title a cluster sample data object
+#' @description A simple data example of a cluster sample, drawn from
+#' Avery and Burkhart Forest Measurements text.
+#' @docType data
+#' @format an object of class dataframe
+#' \itemize{
+#'   \item{clusterID}{cluster}
+#'   \item{volume}{sampled volume - attribute of interest}
+#'   \item{isUsed}{TRUE/FALSE}
+#' }
+#' @name dataPlot
+#' @usage dataPlot
+NULL
+
+#' @title a cluster sample data object
+#' @description A more complex data example of a cluster sample
+#' @docType data
+#' @format an object of class dataframe
+#' \itemize{
+#'   \item{clusterID}{cluster}
+#'   \item{bapa}{basal area per acre - attribute of interest}
+#'   \item{isUsed}{TRUE/FALSE}
+#' }
+#' @name clusterBAData
```

Comment:
Maybe we could change the name to clusterBaData? I thought this was a BAD dataset at first 😛

Reply:
Changed